recipes/google-crc32c/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "google-crc32c" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/googleapis/python-crc32c/archive/v{{ version }}.tar.gz
+  sha256: 83144943a727beaffea6d133a1f9e425b46c95d8940ea429091c1c965d9c7ac7
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
```

You need a compiler:
```suggestion
requirements:
  build:
   - {{ compiler('c') }}
```

---

Thanks for the help